src/modules/keyboardmanager/KeyboardManagerEditorLibrary/BufferValidationHelpers.cpp

```diff
@@ -29,9 +29,11 @@ namespace BufferValidationHelpers
         if (selectedKeyCode != -1)
         {
             // Check if the value being set is the same as the other column
-            if (remapBuffer[rowIndex].first[std::abs(int(colIndex) - 1)].index() == 0)
+            const auto otherColumnIndex = static_cast<size_t>(std::abs(colIndex - 1));
+            const auto& otherColumnKey = remapBuffer.at(rowIndex).mapping.at(otherColumnIndex);
```

Comment:
I don't expect any exception to be thrown. It's a product bug if that happens. Does PowerToys have any guideline on how to handle those unexpected exceptions (send telemetry data, for example)?

Reply:
I don't expect any exception to be thrown. It's a product bug if that happens. Does PowerToys have any guideline on how to handle those unexpected exceptions (send telemetry data, for example)?

Reply:
Yes, we use Windows functionality to obtain anonymized crash dumps, but it's better to prevent crashing in the first place. I guess unhandled exception is better than memory corruption, which could happen previously with `operator[]`. We also have a `InitUnhandledExceptionHandler_x64` which should log them.

Reply:
Thank you for the information. If unhandled exceptions will be eventually handled and logged, I'll keep `.at` to detect errors earlier.